src/SWP/Component/Revision/Repository/RevisionLogRepository.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Templates System.
+ *
+ * Copyright 2015 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Component\Revision\Repository;
+
+use SWP\Bundle\StorageBundle\Doctrine\ORM\EntityRepository;
+
+class RevisionLogRepository extends EntityRepository
```

Comment:
repository should not be placed inside the component because it implements StorageBundle classes, should be moved to the RevisionBundle